ReactWindows/ReactNative/Modules/Toast/ToastModule.cs

```diff
@@ -0,0 +1,114 @@
+﻿using ReactNative.Bridge;
+using System.Collections.Generic;
+using Windows.Data.Xml.Dom;
+using Windows.UI.Notifications;
+
+namespace ReactNative.Modules.Toast
+{
+    using System.Diagnostics;
+    using System.IO;
+    using System.Threading.Tasks;
+    using Map = Dictionary<string, object>;
+    public partial class ToastModule : ReactContextNativeModuleBase
+    {
+        public ToastModule(ReactApplicationContext reactContext)
+            : base(reactContext)
+        {
+        }
+
+        private static readonly string DURATION_SHORT_KEY = "SHORT";
```

Comment:
These could be const.  Also, usually consts / fields are declared before constructor.